src/server/index.ts

```diff
@@ -42,29 +70,7 @@ app.use(
 
 app.use(serverRender());
 
-// eslint-disable-next-line no-unused-vars
```

Comment:
Don't do this. Switching off TS is Tschernobyl-Style engineering.

Reply:
See my previous comment …